capstone/src/main/java/com/google/univiz/api/fix.sh

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
One way to prevent accidentally committing this file is making a `bin` directory inside your home directory. Then you could add this script there and add the bin directory to your `PATH`, which would make `fix.sh` accessible throughout.

Reply:
yep that's a great point because I always end up accidentally committing this :((